source/module_basis/module_ao/parallel_2d.h

```diff
@@ -16,6 +16,8 @@ class Parallel_2D
     Parallel_2D() = default;
     ~Parallel_2D() = default;
 
+    Parallel_2D& operator=(Parallel_2D&& rhs);
```

Comment:
Since the members of this class are either POD types or std::vector, an easier way to implement the move assignment operator is
```suggestion
    Parallel_2D& operator=(Parallel_2D&& rhs) = default;
```

Reply:
Changed. It works. Thanks for your refactor.